systemtest/src/main/java/io/strimzi/systemtest/bases/IMetrics.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.bases;
+
+import io.fabric8.kubernetes.api.model.LabelSelector;
+import io.strimzi.api.kafka.model.KafkaConnectResources;
+import io.strimzi.api.kafka.model.KafkaExporterResources;
+import io.strimzi.api.kafka.model.KafkaResources;
+import io.strimzi.test.executor.Exec;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.strimzi.test.BaseITST.cmdKubeClient;
+import static io.strimzi.test.BaseITST.kubeClient;
+
+public interface IMetrics {
+
+    Logger LOGGER = LogManager.getLogger(IMetrics.class);
+    Object LOCK = new Object();
+
+    /**
+     * Collect metrics from specific pod
+     * @param podName pod name
+     * @return collected metrics
+     */
+    default String collectMetrics(String podName, String metricsPath) throws InterruptedException, ExecutionException, IOException {
+        ArrayList<String> command = new ArrayList<>();
+        command.add("curl");
+        command.add(kubeClient().getPod(podName).getStatus().getPodIP() + ":9404" + metricsPath);
+        ArrayList<String> executableCommand = new ArrayList<>();
+        executableCommand.addAll(Arrays.asList(cmdKubeClient().toString(), "exec", podName, "-n", kubeClient().getNamespace(), "--"));
+        executableCommand.addAll(command);
```

Comment:
This is bizarre! Why not just create `executableCommand = asList(cmdKubeClient().toString(), "exec", podName, "-n", kubeClient().getNamespace(), "--", "curl", kubeClient().getPod(podName).getStatus().getPodIP() + ":9404" + metricsPath);` and be done with it?